torchquad/integration/monte_carlo.py

```diff
@@ -196,6 +196,16 @@ def compiled_integrate(fn, integration_domain):
             # Torch requires explicit tracing with example inputs.
             def do_compile(example_integrand):
                 import torch
+                import warnings
+
+                # The PyTorch trace compilation warnings contain many false
+                # positives, so we hide all trace compiler warnings
+                def trace_without_warnings(*args, **kwargs):
+                    with warnings.catch_warnings():
+                        warnings.filterwarnings(
+                            "ignore", category=torch.jit.TracerWarning
+                        )
+                        return torch.jit.trace(*args, **kwargs)
```

Comment:
I guess you are not making this function a dedicated file in the `utils` folder to avoid the torch import if possible, right?

Reply:
I've moved it to the `utils.py` file now, which also contains helper functions.